### ISSUE TITLE
Crash with: TypeError: Cannot read property 'getJsDocTags' of undefined

### ISSUE THREAD

**Issue by CyrusNajmabadi** [aws/jsii#581]
This code:

```ts
export class HelloJsii {
    public strings: string[];

    constructor() {
        this.strings = ["a", "c", "c"];
    }

    public theStrings() { return this.strings };
    public addString(s: string) {
        this.strings.push(s);
    }
}
```

Produces this error: [2019-07-02T10:35:32.937] [ERROR] jsii/compiler - Error during type model analysis: TypeError: Cannot read property 'getJsDocTags' of undefined

**Comment by ulle** [comment]
Ran into the same issue. Seems to be triggered by dangling ;s behind the method definition.

**Comment by aripalo** [comment]
Yeah, can confirm. Had the same error and removing the dangling semicolons solved it.

**Comment by eladb** [comment]
Same issue here. Pretty bad experience, especially given there is no error information.

**Comment by RomainMuller** [comment]
This was fixed in #2163, which will be released very soon!
